qa/automation/CareerPortal.Tests.Scenarios/2. Extended example/homepage/name_field_validation/email_field_validation/Send_cv/AtachCv.feature

```diff
@@ -0,0 +1,131 @@
+Feature: 1.1.3 Attach CV and Send 
+
+Background: 
+
+	Given User  created 7 byte  "cv1.pdf" file
+	And User created 10 Megabyte  "cv2.pdf" file
+	And user created 7 byte  "cv3.docx" file
+	And user created 10 megabyte "cv4.docx" file
+	And user created 7 byte"cv5.rtf" file
+	And User created 10 megabyte "cv6.rtf" file
+	And User created 7 byte "cv7.doc" file
+	And User created 10 megabyte "cv8.doc" file
+	And User created 10.1 megabyte "cv9.doc" file
+	And User on https://qa-career.quantori.com/ru website
+	And User scrolls to the end of the pege
+	And User fills in <name_field> "Timur Abdraimov"
+	And User fills in <email_field> "timurabdraimov4@gmail.com"
+	
+
+
+
+	@Negative
+	Scenario: Sending cv without Attaching file
+	When User clicks "<Send> Send " Button
+	Then System displayed error "The file is required"
+	
+
+
+@Positive
+@Equivalence_partition
+Scenario: 1.1.10 Attaching "cv1.pdf" file
```

Comment:
All positive scenarios regarding the file of CV should be united in one Scenario Outline (review example above)
All negative scenarios regarding file of CV also can be united in one Scenario Outline 

This comment is valid for all similar mistakes below.